spec/helpers/regions_helper_spec.rb

```diff
@@ -31,14 +31,14 @@
 
     it 'renders the region with a wrapper if a tag is specified' do
       content = render_region(:body, tag: :section)
-      content.should have_selector('h1', content: 'Fred Flinstone')
+      content.should have_selector('h1', text: 'Fred Flinstone')
       content.should have_selector("section[data-mercury='full']")
       content.should have_selector('section#body')
     end
 
     it 'renders the region with a wrapper and set class and region type from options' do
       content = render_region(:body, tag: :section, html: {class: 'blah'}, region_type: 'snippets')
-      content.should have_selector('h1', content: 'Fred Flinstone')
+      content.should have_selector('h1', text: 'Fred Flinstone')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.